### ISSUE TITLE
Remove try-catch from target single LogEventInfo Write override

### ISSUE THREAD

**Issue by snakefoot** [loresoft/NLog.Mongo#14]
Remove try-catch exception-handler from this method:

`protected override void Write(LogEventInfo logEvent)`

It currently breaks the NLog error-handling with possible retry or fallback, as the exception is silently ignored. The exception should flow to the NLog-engine, that will handle it.

The above rule does not apply for the `Write(AsyncLogEventInfo[])`, which is implemented perfectly (As you are correctly calling the `Continuation`-method)

**Comment by pwelter34** [comment]
Do you have an example or perhaps a pull request?

**Comment by snakefoot** [comment]
Example of how it should look:

```
        protected override void Write(LogEventInfo logEvent)
        {
            var document = CreateDocument(logEvent);
            var collection = GetCollection();
            collection.InsertOne(document);
        }
```

**Comment by snakefoot** [comment]
Resolved by 34e79b26d4d838263962ba8766f71cb8e2cb5c18
